.github/workflows/ci.yml

```diff
@@ -19,11 +19,6 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - uses: pnpm/action-setup@v4
-        with:
-          version: 7
-      - uses: actions/setup-node@v4
-        with:
-          node-version: 18
```

Comment:
why are we getting rid of this?

Reply:
It's a redudant setup action. Not necessary to have node setup with pnpm setup action. The version for pnpm is now specified in package.json as well.